sql/core/src/main/scala/org/apache/spark/sql/execution/ui/AllExecutionsPage.scala

```diff
@@ -202,100 +385,127 @@ private[ui] abstract class ExecutionTable(
         </td>
       }}
       {if (showSucceededJobs) {
-        <td>
-          {jobLinks(JobExecutionStatus.SUCCEEDED)}
-        </td>
-      }}
+      <td>
+        {jobLinks(JobExecutionStatus.SUCCEEDED)}
+      </td>
+    }}
       {if (showFailedJobs) {
-        <td>
-          {jobLinks(JobExecutionStatus.FAILED)}
-        </td>
-      }}
+      <td>
+        {jobLinks(JobExecutionStatus.FAILED)}
+      </td>
+    }}
     </tr>
   }
 
-  private def descriptionCell(
-      request: HttpServletRequest,
-      execution: SQLExecutionUIData): Seq[Node] = {
+  private def descriptionCell(execution: SQLExecutionUIData): Seq[Node] = {
     val details = if (execution.details != null && execution.details.nonEmpty) {
-      <span onclick="clickDetail(this)" class="expand-details">
+      <span onclick="this.parentNode.querySelector('.stage-details').
+      classList.toggle('collapsed')"
```

Comment:
Nit: pull this onto previous line? the indent is odd, at least

Reply:
Done. Thanks.